modules/check_in/spec/services/v2/chip/client_spec.rb

```diff
@@ -103,7 +103,7 @@
           demographicsNeedsUpdate: true,
           demographicsConfirmedAt: '2021-11-30T20:45:03.779Z',
           nextOfKinNeedsUpdate: true,
-          nextOfConfirmedAt: '2021-11-30T20:45:03.779Z',
+          nextOfKinConfirmedAt: '2021-11-30T20:45:03.779Z',
```

Comment:
Do we need to explicitly enable the `check_in_experience_chip_service_nok_confirmation_update_enabled` feature flag in a `before` block for this?

Reply:
I think this is just an example demographics structure that we're passing into the Client instances pre_check_in method. In both test cases, I don't think we're making use of the actual keys or values within. The entire hash just gets turned into a JSON and we're testing a couple of outcomes. We could technically pass in a blank hash and I think the test cases will pass without any issues. However, I think it would be nice if we can pass in a fully qualified structure that illustrates to the user what the contents may actually be.